student.rb

```diff
@@ -8,10 +12,6 @@ def initialize(options = {})
     @name = options.fetch(:name, "")
   end
 
-  def offer_high_five
-    "High five!"
-  end
-
   def set_phase(num)
     response = ""
```

Comment:
There are still some common methods like name, age, phase between classes Student and Educator. How can you refactor this?